checkbox-support/checkbox_support/dbus/gnome_monitor.py

```diff
@@ -104,6 +106,107 @@ def set_extended_mode(self) -> Dict[str, str]:
         self._apply_monitors_config(state[0], extended_logical_monitors)
         return configuration
 
+    def cycler(
```

Comment:
I would just call it cycle, which is the action taken by this function. `cycler` sounds more like a class name to me.
```suggestion
    def cycle(
```